tests/VolleyManagement.UnitTests/WebApi/Controllers/TournamentControllerTests.cs

```diff
@@ -1,24 +1,26 @@
-﻿namespace VolleyManagement.UnitTests.WebApi.Controllers
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Web.Http.Results;
+using VolleyManagement.Contracts;
+using VolleyManagement.Crosscutting.Contracts.Extensions;
+using VolleyManagement.Domain.GameReportsAggregate;
+using VolleyManagement.Domain.GamesAggregate;
+using VolleyManagement.Domain.TournamentsAggregate;
+using VolleyManagement.UI.Areas.WebApi.Controllers;
+using VolleyManagement.UI.Areas.WebApi.ViewModels.Tournaments;
+using VolleyManagement.UnitTests.Services.GameReportService;
+using VolleyManagement.UnitTests.Services.GameService;
+using VolleyManagement.UnitTests.Services.TournamentService;
+using VolleyManagement.UnitTests.WebApi.Standings;
+using VolleyManagement.UnitTests.WebApi.ViewModels;
+using TournamentViewModelComparer = VolleyManagement.UnitTests.WebApi.ViewModels.TournamentViewModelComparer;
+
+namespace VolleyManagement.UnitTests.WebApi.Controllers
 {
```

Comment:
revertaty 
![image](https://user-images.githubusercontent.com/17709640/36846136-1cb9ed32-1d62-11e8-8811-fdf1bca36ea6.png)

Reply:
why?

Reply:
namespace is on line 1 and ALL using after, entire solution has that rule

Reply:
*ALL*

Reply:
`ALL`